File: sdk/SDK.Examples/src/DownloadDocumentsExample.cs
using System;
using System.IO;
using Silanis.ESL.SDK;
using Silanis.ESL.SDK.Builder;

namespace SDK.Examples
{
	public class DownloadDocumentsExample
	{
		public static string apiToken = "YOUR TOKEN HERE";
		public static string baseUrl = "ENVIRONMENT URL HERE";

		public static void Main (string[] args)
		{
			// Create new esl client with api token and base url
			EslClient client = new EslClient (apiToken, baseUrl);
			PackageId packageId = new PackageId ("GLK2xasqLvFe2wc4qwO5iTKyjx42");

            byte[] downloadedDocument = client.DownloadDocument (packageId, "testing");
            File.WriteAllBytes (Directory.GetCurrentDirectory() + "/downloaded.pdf", downloadedDocument);

            byte[] downloadedEvidence = client.DownloadEvidenceSummary (packageId);
            File.WriteAllBytes (Directory.GetCurrentDirectory() + "/evidence-summary.pdf", downloadedEvidence);

            byte[] downloadedZip = client.DownloadZippedDocuments (packageId);
            File.WriteAllBytes (Directory.GetCurrentDirectory() + "/package-documents.zip", downloadedZip);
		}
	}
}

File: sdk/SDK.VBExamples/SDK.VBExamples/DownloadDocumentsExample.vb
Imports System
Imports System.IO
Imports Silanis.ESL.SDK
Imports Silanis.ESL.SDK.Builder

Namespace SDK.VBExamples
    Public Module DownloadDocumentsExample
        Dim apiToken = "YOUR TOKEN HERE"
        Dim baseUrl = "ENVIRONMENT URL HERE"

        Sub Main()
            ' Create new esl client with api token and base url
            Dim client = New EslClient(apiToken, baseUrl)
            Dim packageId = New PackageId("GLK2xasqLvFe2wc4qwO5iTKyjx42")

            Dim documentContent = client.DownloadDocument(packageId, "testing")
            File.WriteAllBytes(Directory.GetCurrentDirectory() + "/downloaded.pdf", documentContent)

            Dim evidenceContent = client.DownloadEvidenceSummary(packageId)
            File.WriteAllBytes(Directory.GetCurrentDirectory() + "/evidence-summary.pdf", evidenceContent)

            Dim zipContent = client.DownloadZippedDocuments(packageId)
            File.WriteAllBytes(Directory.GetCurrentDirectory() + "/package-documents.zip", zipContent)
        End Sub
    End Module
End Namespace